src/libcollections/slice.rs

```diff
@@ -1087,7 +1086,7 @@ impl<T> [T] {
     /// elements.
     ///
     /// This sort is stable and `O(n log n)` worst-case but allocates
-    /// approximately `2 * n`, where `n` is the length of `self`.
+    /// approximately `n / 2`, where `n` is the length of `self`.
```

Comment:
There's still a constant factor hidden in this wording, because it doesn't contain any units. `n / 2` *what*? Bytes? Words? `size_of::<T>()`?

The old version also has this problem, but while you're updating it and have the details fresh in mind, could you add this detail?